RELEASE.md

```diff
@@ -0,0 +1,304 @@
+
+    $ git checkout master
+    $ git pull --rebase --tags origin master
+
+Create an RC branch: If there isn't yet a {MAJOR}.{MINOR}.x branch
+corresponding to the release (e.g. 1.7.x), create one from master.
+
+    $ git checkout -b 1.7.x
+
+
+## 2. Update CHANGELOG.md
+
+Make sure the top of [CHANGELOG.md](../CHANGELOG.md) has a section
+for the upcoming release. See previous releases in CHANGELOG.md for how
+to structure and format the release notes.
+
+For our fix and enhancement PRs we typically write the corresponding changelog
+entry as part of that PR, but for PRs that have been submitted from the
+community we typically need to add changelog entries afterwards.
+
+To find all changes since the last release, do:
+
+    $ git log v0.11.3..    # the last release
+
+Go through the printed git history and identify and changes that are not
+already in the changelog and that are relevant to the end-users of the client.
+For example, a bug fix for a bug that users would be seeing should have a
+corresponding changelog entry, but a fix to the build system is typically not
+interesting and can be skipped.
+
+For commits provided by the community we need to provide proper attribution
+by looking up the GitHub username for each such change and then mention
+the user from the changelog entry, e.g.:
+
+` * Added OAuth support to AdminClient ([someghuser](https://github.com/someghuser))`
+
+The changelog must also say which librdkafka version is bundled, a link to the
+librdkafka release notes, and should also mention some of the most important
+changes to librdkafka in that release.
+
+*Note*: `CHANGELOG.md` is in MarkDown format.
+
+
+## 3. Download librdkafka config file
+
+    $ cd src/ConfigGen
+    $ dotnet run v0.11.x-RC1
+    $ mv out/Config_gen.cs ../Confluent.Kafka
+
+
+## 4. Update librdkafka version
+
+Search / replace to the old/new version number in:
+    - C# comments
+    - The change log
+    - The reademe
+    - csproj files (librdkafka.redist version and .net library versions)
+
+*Note*: Make sure librdkafka.redist is use the latest version.
+
+
+## 5. Run tests
+
+*Note*: Make sure .net core 2.1 installed.  
+
+    - Confluent.Kafka.IntegrationTests
+    - Confluent.Kafka.UnitTests
+    - Confluent.SchemaRegistry.IntegrationTests
+    - Confluent.SchemaRegistry.Serdes.IntegrationTests
+    - Confluent.SchemaRegistry.UnitTests
+    - Confluent.SchemaRegistry.Serdes.UnitTests
+
+Cd to the above folders and run:
+    $ dotnet test -f netcoreapp2.1
+
+Start a single broker Kafka cluster on localhost and an
+instance of Schema Registry (default configs are fine). There are some
+tests for auth, and a docker file which is helpful for that, but verification
+for these is usually not needed.
+
+
+## 6. Tag, CI build, wheel verification, upload
+
+The following sections are performed in multiple iterations:
+
+ - **TEST ITERATION** - a number of test/release-candidate iterations where
```

Comment:
i never need to do this. only the unit tests are run in CI, and they never fail there, if they pass locally. remove this section.